tests/unittest/db/test_helpers.py

```diff
@@ -0,0 +1,169 @@
+        )
+        model_2 = Object(
+            aid="aid2",
+            oid="oid2",
+            lastmjd="lastmjd",
+            firstmjd="firstmjd",
+            meanra=100.0,
+            meandec=50.0,
+            ndet=5,
+            probabilities=[
+                {
+                    "classifier_name": "lc_classifier",
+                    "classifier_version": "lc_classifier_1.0.0",
+                    "class_name": "CLASS1",
+                    "probability": 0.4,
+                    "ranking": 2,
+                },
+                {
+                    "classifier_name": "lc_classifier",
+                    "classifier_version": "lc_classifier_1.0.0",
+                    "class_name": "CLASS2",
+                    "probability": 0.6,
+                    "ranking": 1,
+                },
+            ],
+        )
+        self.obj_collection.insert_many([model_1, model_2])
+
+
+    def test_error_input_length(self):
+        with self.assertRaises(Exception):
+            create_of_update_probabilities(self.mongo_connection, "aid1", "classifier1", "v1", ["class1", "class2"], [0.1, 0.2, 0.3])
+
+
+    def test_create_probabilities(self):
+        self.create_2_objects()
+
+        create_of_update_probabilities(
+            self.mongo_connection,
+            "aid2",
+            "stamp_classifier",
+            "stamp_classifier_1.0.0",
+            ["CLASS1", "CLASS2"],
+            [0.3, 0.7]
+        )
+
+        f1 = self.obj_collection.find_one({"aid": "aid2"})
+
+        expected_probabilities = [
+            {
+                "classifier_name": "lc_classifier",
+                "classifier_version": "lc_classifier_1.0.0",
+                "class_name": "CLASS1",
+                "probability": 0.4,
+                "ranking": 2,
+            },
+            {
+                "classifier_name": "lc_classifier",
+                "classifier_version": "lc_classifier_1.0.0",
+                "class_name": "CLASS2",
+                "probability": 0.6,
+                "ranking": 1,
+            },
+            {
+                "classifier_name": "stamp_classifier",
+                "classifier_version": "stamp_classifier_1.0.0",
+                "class_name": "CLASS2",
+                "probability": 0.7,
+                "ranking": 1,
+            },
+            {
+                "classifier_name": "stamp_classifier",
+                "classifier_version": "stamp_classifier_1.0.0",
+                "class_name": "CLASS1",
+                "probability": 0.3,
+                "ranking": 2,
+            },
+        ]
+
+        self.assertEqual(f1["probabilities"], expected_probabilities)
+
+    def test_update_probabilities(self):
+        self.create_2_objects()
+        
+        create_of_update_probabilities(
+            self.mongo_connection,
+            "aid1",
+            "stamp_classifier",
+            "stamp_classifier_1.0.0",
+            ["CLASS1", "CLASS2"],
+            [0.3, 0.7]
+        )
```

Comment:
Could we have the object aid1 also come with probabilities for another classifier? just to make sure only the intended modifications are made and that it doesn't mess with other existing ones?